content/en/docs/components/pipelines/pipeline-root.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "Pipeline Root"
+description = "Getting started with Kubeflow Pipelines pipeline root"
+weight = 50
+
++++
+{{% beta-status
+feedbacklink="https://github.com/kubeflow/pipelines/issues" %}}
+
+Starting from [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/sdk/v2/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports a new intermediate artifact repository feature: pipeline root in both [standalone deployment](https://www.kubeflow.org/docs/components/pipelines/installation/standalone-deployment/) and [AI platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs).
+
+## Before you start
+This guide tells you the basic concepts of Kubeflow Pipelines pipeline root and how to use it.
+This guide assumes that you already have Kubeflow Pipelines installed, or want to use standalone or AI platform Pipelines options in the [Kubeflow Pipelines deployment
+guide](/docs/components/pipelines/installation/) to deploy Kubeflow Pipelines.
+
+## What is pipeline root?
+
+Pipeline root represents an artifact repository where Kubeflow Pipelines stores a pipeline's artifacts.
+This feature supports MinIO, S3, GCS natively using [Go CDK](https://github.com/google/go-cloud). Artifacts can be more accessible in S3 and GCS when integrating Kubeflow Pipelines with other systems.
+
+**Note:** For MinIO, you can't change the MinIO instance. Kubeflow Pipelines can only use the Minio instance deployed with itself.
+
+## How to configure pipeline root authentication 
+#### MinIO
+You don't need to pass the authentication for MinIO
+
+#### GCS
+If you want to specify the `pipeline root` to GCS :
+
+* Set up pipeline root authentication in pipeline level via sdk:
+`dsl.get_pipeline_conf().add_op_transformer(aws.use_gcp_secret('xxx', ‘xxx’, ‘xxx’))`
+
+* Set up pipeline root authentication via service account key:
+check [authentication-pipelines](https://www.kubeflow.org/docs/distributions/gke/pipelines/authentication-pipelines/)
```

Comment:
Note, these are optional, users can authenticate to GCP using any of the 3 supported methods.
See existing documentation: https://www.kubeflow.org/docs/distributions/gke/pipelines/authentication-pipelines/.

and our recommendation is actually the other two methods -- if not separation sensitive, use compute default service account. If separation sensitive, use workload identity. service account key is deprecated.

Therefore, my personal suggestion would be directly referring to the documentation https://www.kubeflow.org/docs/distributions/gke/pipelines/authentication-pipelines/ and say users need to authenticate to google cloud using any of 3 documented ways (rough ideas).

Reply:
done